### ISSUE TITLE
A blank window appears at startup.

### ISSUE THREAD

**Issue by marvelph** [Jaysce/Spaceman#40]
It seems that a blank window appears each time Spaceman is started.
The title of the window is “Spaceman Settings”.

Spaceman 1.0
macOS 15.0.1
![image](https://github.com/user-attachments/assets/80386062-13bb-4258-b28c-95448cf2835c)

**Comment by jsonMartin** [comment]
Yeah, this is also happening for me. Otherwise, the app seems to work just fine, but it's annoying to have to close this every time on Startup.

I'm using MacOS Sequoia 15.1 (24B83)

**Comment by erikmartinessanches** [comment]
I have the same issue, macOS 15.1, M1 Pro. It appears the app is not running as it was for jsonMartin. I am a first-time user.

**Comment by yunruse** [comment]
Can confirm. I think it is a Sequoia thing. It's also identical to an issue I've seen that has been recently fixed:

- DamascenoRafael/reminders-menubar#204

@DamascenoRafael, what behaviour caused this previously?

**Comment by obfischer** [comment]
Same here

**Comment by DamascenoRafael** [comment]
@yunruse, I just compiled the application again using a mac running Sequoia. If it was some change in the code itself that fixed this, I didn't notice.

**Comment by usagimaru** [comment]
Can this bug be fixed by removing `EmptyView()` from the Settings described in `SpacemanApp` in AppDelegate?

```
var body: some Scene {
	Settings {
		EmptyView() // REMOVE this line
	}
}
```

https://github.com/Jaysce/Spaceman/blob/17fd96b3c0cf3cf0dfe6bbe44eb2725c92ff6b61/Spaceman/AppDelegate.swift#L48-L54

I found a similar problem and solution on Stack Overflow.

MacOS Swift (UI): Prevent App Window Opening
https://stackoverflow.com/questions/79068815/macos-swift-ui-prevent-app-window-opening?rq=2

**Comment by wessamfathi** [comment]
Sadly, Spaceman doesn't work for me as well (M4/Sequoia). I get the empty screen on the first run, and that's it. I cannot see or use Spaceman in any way.

**Comment by pinguinpfleger** [comment]
Sadly, Spaceman doesn't work for me as well (M4/Sequoia). I get the empty screen on the first run, and that's it. I cannot see or use Spaceman in any way.

**Comment by szechp** [comment]
this project seems abandoned, but there's an updated fork:

https://github.com/ruittenb/Spaceman

@Jaysce is this project abandoned?

**Comment by gregorias** [comment]
> this project seems abandoned, but there's an updated fork:
> 
> https://github.com/ruittenb/Spaceman

Last time I checked in 2025, that fork also had this blank window issue.

**Comment by erikmartinessanches** [comment]
> > this project seems abandoned, but there's an updated fork:
> > https://github.com/ruittenb/Spaceman
> 
> Last time I checked in 2025, that fork also had this blank window issue.
I am running Spaceman on M1 Pro macOS 15.5. The blank window appeared only once at the first launch for me, after that, I was able to access its preferences.

**Comment by gregorias** [comment]
> after that, I was able to access its preferences.

Accessing preferences is not what this issue is about. It’s about showing the blank window on startup, which the fork has not fixed.

**Comment by AlexGrafe** [comment]
I now use this: https://apps.apple.com/de/app/desktop-space-renamer/id6745852577?l=en-GB&mt=12
Costs 10€ but does the job nicely.

**Comment by ruittenb** [comment]
@jensb when you say "Most recent release", do you mean 1.13.5?
